docs/en/interfaces/formats.md

```diff
@@ -1210,6 +1211,65 @@ SELECT * FROM json_each_row_nested
 - [output_format_json_array_of_rows](../operations/settings/settings.md#output_format_json_array_of_rows) - output a JSON array of all rows in JSONEachRow(Compact) format. Default value - `false`.
 - [output_format_json_validate_utf8](../operations/settings/settings.md#output_format_json_validate_utf8) - enables validation of UTF-8 sequences in JSON output formats (note that it doesn't impact formats JSON/JSONCompact/JSONColumnsWithMetadata, they always validate utf8). Default value - `false`.
 
+## BSONEachRow {#bsoneachrow}
+
+In this format, ClickHouse formats/parses each row as a separated BSON Document.
```

Comment:
Is it (de)serializss sequence of documents that goes back to back without any separator? It's worth mentioning. For me, "row" sounds like something terminated by '\n' like in any text format.

Reply:
Without any separator, right. Agree, I will mention that in docs